sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -655,6 +675,9 @@ def analyze_sentiment(  # type: ignore
             :class:`~azure.ai.textanalytics.SentenceSentiment` objects
             will have property `mined_opinions` containing the result of this analysis. Only available for
             API version v3.1-preview and up.
+        :keyword str string_index_type: Specifies the method used to interpret string offsets.  Possible values are
```

Comment:
Not sure why this is showing as added here.  This should've been added in a previous PR and merged to master already.

Reply:
Yes. KeyPhrases Extraction should not has string_index_type

Reply:
Actually if you expand the context preceding this it looks like this is for Sentiment Analysis, not Key Phrases. Maybe I forgot to add them in my previous PR.